knowledge-base/objectdatasource-assembly-not-recognized-by-standalone-designer.md

```diff
@@ -0,0 +1,42 @@
+---
+title: ObjectDataSource Assembly Not Recognized by the Standalone Designer
+description: How to troubleshoot which assemblies to add to the Standalone designer folder when an ObjectDataSource is not property recognized
```

Comment:
How to troubleshoot which assemblies to add to the Standalone Designer folder when an ObjectDataSource assembly types are missing in the wizard.